yarn.lock

```diff
@@ -4,31 +4,31 @@
 
 "@assemblyscript/loader@^0.9.4":
   version "0.9.4"
-  resolved "https://registry.yarnpkg.com/@assemblyscript/loader/-/loader-0.9.4.tgz#a483c54c1253656bb33babd464e3154a173e1577"
+  resolved "https://registry.npmjs.org/@assemblyscript/loader/-/loader-0.9.4.tgz"
```

Comment:
I wonder whats up with these changes. Most likely nothing to worry about, though! (And definitely doesn't prevent merging :)

Reply:
Yeah, that's odd. I sometimes use `npm` instead of `yarn` out of habit, but doesn't seem like that would update `yarn.lock`. :thinking: